src/api/pureHelpers.ts

```diff
@@ -69,45 +40,67 @@ export const groupPositionsArray = (
         item.insuranceStart = val.insuranceStart;
         item.coverageDecPercent = val.coverageDecPercent;
         item.fullCoverage = val.fullCoverage;
+        item.pendingReserveReward = val.pendingReserveReward;
 
         const sumStakeAmount = filtered
           .map(x => Number(x.stake.amount || 0))
           .reduce((sum, current) => sum + current);
-        const sumStakeUsd = filtered
-          .map(x => Number(x.stake.usdValue || 0))
-          .reduce((sum, current) => sum + current);
 
-        const sumProtectedValueAmount = filtered
+        const sumFullyProtected = filtered
           .map(x => Number(x.fullyProtected ? x.fullyProtected.amount : 0))
           .reduce((sum, current) => sum + current);
-        const sumProtectedValueUsd = filtered
-          .map(x => Number(x.fullyProtected ? x.fullyProtected.usdValue : 0))
-          .reduce((sum, current) => sum + current);
+        let sumFullyProtectedWithReward: BigNumber;
 
         const sumProtectedAmount = filtered
           .map(x => Number(x.protectedAmount ? x.protectedAmount.amount : 0))
           .reduce((sum, current) => sum + current);
-        const sumProtectedAmountUsd = filtered
-          .map(x => Number(x.protectedAmount ? x.protectedAmount.usdValue : 0))
-          .reduce((sum, current) => sum + current);
+        let sumProtectedWithReward: BigNumber;
+
+        if (symbol === "BNT") {
+          sumFullyProtectedWithReward = item.pendingReserveReward.plus(
+            sumFullyProtected
+          );
+          sumProtectedWithReward = item.pendingReserveReward.plus(
+            sumProtectedAmount
+          );
+        } else {
+          const bntRewardUsd = item.pendingReserveReward.times(
+            val.bntTokenPrice
+          );
+          sumFullyProtectedWithReward = bntRewardUsd
+            .div(val.reserveTokenPrice)
+            .plus(sumFullyProtected);
+          sumProtectedWithReward = bntRewardUsd
+            .div(val.reserveTokenPrice)
+            .plus(sumProtectedAmount);
+        }
+
+        const sumFullyProtectedWithRewardUSD =
+          Number(sumFullyProtectedWithReward) * val.reserveTokenPrice;
+
+        const sumProtectedWithRewardUSD =
+          Number(sumProtectedWithReward) * val.reserveTokenPrice;
+
         const sumFees = filtered
           .map(x => Number(x.fees ? x.fees.amount : 0))
           .reduce((sum, current) => sum + current);
```

Comment:
```suggestion
          .reduce((sum, current) => sum + current, 0);
```

Lets just add the initial value, I've seen instances where the reduce method is used for Maths but there's a mistake made and you end up building a string instead given strings also support ```+``` operators. Makes it that tiny bit clearer to the reader too.